package.json

```diff
@@ -20,12 +20,12 @@
     "url" : "https://github.com/getditto/react-ditto.git"
   },
   "peerDependencies": {
-    "@dittolive/ditto": "^1.0.17",
+    "@dittolive/ditto": "1.1.5",
```

Comment:
This peer dependency is too restrictive, I think we should allow for higher compatible versions too by using `^` here.